app/assets/stylesheets/citation.scss

```diff
@@ -0,0 +1,142 @@
+.citation {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 14px;
+  font-size: .875rem;
+  line-height: 1.1429;
+
+  @include govuk-media-query($from: tablet) {
+    font-size:16px;
+    font-size: 1rem;
+    line-height: 1.25;
+
+    a {
+      font-family: sans-serif;
+    }
+
+    &--inverse-padded .citation__list {
+      margin: 15px;
+    }
+
+    &__term {
+      box-sizing:border-box;
+      float: left;
+      clear: left;
+      padding-right: 5px;
+      margin-top: 0;
+    }
+
+    &__term .citation__definition {
+      line-height: 1.4;
+    }
+
+    &__definition:not(:last-of-type) {
+      margin-bottom: 5px;
+    }
+  }
+
+  a {
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-decoration: underline;
+    font-weight: bold;
+    color: govuk-colour('black');
+  }
+
+  .citation::after {
+    content: '';
+    display: block;
+    clear: both;
+  }
+
+  .direction-rtl {
+    direction: rtl;
+    text-align: start;
+  }
+
+  &-inverse {
+    color: govuk-colour('white');
+
+    a:link,
+    a:hover,
+    a:visited,
+    a:active {
+        color: govuk-colour('white');
+      }
+
+    a:focus {
+      color: govuk-colour('black');
+    }
+
+    &-padded {
+      padding: 10px;
+    }
+
+    &-padded &.citation__list {
+      margin: 10px;
+    }
+  }
+
+  &__definition {
+    margin: 0;
+  }
+
+  &__toggle-wrap {
+    display: none;
+  }
+
+}
+
+.citation .citation__definition-link {
+  font-weight: normal;
+}
+
+.citation.direction-rtl .citation__definition {
```

Comment:
`direction-rtl` seems unlikely this is even relevant, it is also not used in the HTML.